Sources/CubeFoundationSwiftUI/Text/CustomFont.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CustomFont.swift
+//  CubeFoundation
+//
+//  Created by Ben Shutt on 19/05/2023.
+//  Copyright © 2023 3 SIDED CUBE APP PRODUCTIONS LTD. All rights reserved.
+//
+
+import SwiftUI
+
+/// Types of font
+public enum CustomFont: ExpressibleByStringLiteral, Equatable, Hashable {
```

Comment:
Instead of the extra thing, is it worth making `Font.Name` the `enum` to do what this one does? 🤔